src/components/MovieForm/MovieForm.tsx

```diff
@@ -0,0 +1,105 @@
+import './MovieForm.css'
+import React from 'react'
+import { MovieGenres } from '../../common/constants/constants.ts'
+
+type MovieFormProps = {
+    initialInfo?: {
+        title: string
+        releaseDate: string
+        movieUrl: string
+        rating: number
+        runtime: number
+        overview: string
```

Comment:
I think you still need `genres` in `initialInfo`. Consider the case when you are editing an existing movie - you'd want  to highlight the chosen options in the whole list of possible genres.